student/views/info.py

```
@@ -0,0 +1,61 @@
+        summary="경기대학교 학생 정보",
+        description="학교 시스템에 로그인하여 경기대학교 학생 정보를 불러옵니다.",
+        request=LogInSerializer,
+        responses={
+            status.HTTP_200_OK: StudentInformationResponseSerializer,
+        },
+    )
+    def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        studentId = serializer.validated_data.get("studentId")
```

Pylint (3.3.7)

[convention] Variable name "studentId" doesn't conform to snake_case naming style

(C0103)